requirements.txt

```diff
@@ -15,3 +15,4 @@ requests>=2.20.0
 WTForms==2.2.1
 APScheduler==3.6.1
 passlib==1.7.1
+paver==1.3.4
```

Comment:
Why don't we remove Paver and `pavement.py` alltogether?

Reply:
I managed to convert all cli commands to `click`, so there is no technical reason to keep `paver`.

I don't know if other people made their own docker scripts/workflow and I thought it would be good to keep it around, but mark it as deprecated so we can remove it the next version.

Reply:
In this case folks can use an earlier version of GHC no?

Reply:
I would suggest to remove Paver.

Reply:
I removed paver with commit 57383c0